### ISSUE TITLE
Bug or feature: Capitalized and non-capitalized are sorted separately & feature: allow for frontmatter sets

### ISSUE THREAD

**Issue by ksdavidc** [mariomui/obsidian-sort-frontmatter#3]
It appears that capitalized fields and non-capitalized fields are sorted separately.

`ABCabc`
instead of 
`AaBbCc`
or  Reversed 
`aAbBcC`

Not sure if this is a feature or a bug, as in my case, I can use it to create 2 sets of fields, but I can imagine it is not always desirable, or could lead to conflicts.

This brings up the idea (probably in the not planned category) of what if somehow relevant fields could be parsed separately.

**Comment by mariomui** [comment]
DOC_VERSION: v0.0.0
MACRO_VERSION: v0.0.4
MACRO_SOURCE: "xxx"
interim_field: v0.0.1

This is by design. So the capitalized versions are shoved to the top. 
* Reasoning: I experiment with my note templates often. ALL-CAPS differentiates the fields that are battle tested and useful from the interim fields, which are temporary affairs.

I also use it to create two sets of fields. 
It doesn't look you have a need for a different sorting behavior option. 

TLDR, works as intended. <-- (lol i actually mean it this time)

P.S. I sortah designed the api with a strategy pattern in mind for the sorting behavior. 

https://github.com/mariomui/obsidian-sort-frontmatter/blob/15223c4fb07d25b3e3de8772820f8fcaf4c49356/src/main.ts#L46C18-L46C58

The third parameter is a custom sortby function taking 2 parameters. 

https://github.com/mariomui/obsidian-sort-frontmatter/blob/master/src/main.ts#L46C18-L46C58
Feel free to submit a sortby strategy.
